test/index.js

```diff
@@ -1,102 +1,125 @@
-import parse from '../lib/index.js';
-import {expect} from 'chai';
-import assert from 'assert';
-import { dirname } from 'path';
-import { fileURLToPath } from 'url';
+import parse from "../lib/index.js";
```

Comment:
Do not include these changes in the PR. We are not linting/reformatting code in this PR.